game/songparser.cc

```diff
@@ -70,16 +70,17 @@ SongParser::SongParser(Song& s): m_song(s) {
 			type = Type::XML;	// XMLPP should deal with encoding so we don't have to.
 		}
 		else {
-			UnicodeUtil::convertToUTF8(m_ss.str(), s.filename.string());
-			if (smCheck (m_ss.str())) {type = Type::SM; } else if (txtCheck (m_ss.str())) {
+			std::string ss = UnicodeUtil::convertToUTF8(m_ss.str(), s.filename.string());
+			if (smCheck (ss)) {type = Type::SM; } else if (txtCheck (ss)) {
```

Comment:
I would like to see some line breaks ;-)

Reply:
Not saying I won't add them but man I seriously want to know what's wrong with your editor width setting 😂